scalafmt-tests/src/test/resources/scala3/OptionalBraces.stat

```diff
@@ -2208,3 +2208,35 @@ def foo2: Int =
       44
    end if
 end foo2
+<<< 2790 for loops indentation
+def update(tpf: Float): Unit =
+  val s = stateModifier(currentState, tpf)
```

Comment:
optional: i don't think we need these three lines. just the loops is probably enough.